lib/wor/requests/request_error.rb

```diff
@@ -1,11 +1,5 @@
 module Wor
   module Requests
-    class RequestError < StandardError
-      attr_reader :response
-
-      def initialize(response)
-        @response = response
-      end
-    end
+    class RequestError < StandardError; end
```

Comment:
We need the response in the exception, otherwise the service cannot retrieve any information from the the failed request